src/themes/saas/components/modal.ts

```diff
@@ -0,0 +1,101 @@
+import { modalAnatomy as parts } from '@chakra-ui/anatomy'
+import { createMultiStyleConfigHelpers } from '@chakra-ui/styled-system'
+
+const { defineMultiStyleConfig } = createMultiStyleConfigHelpers(parts.keys)
+
+export const Modal = defineMultiStyleConfig({
+  baseStyle: {
+    overlay: {
+      bgColor: 'rgba(0 ,0 ,0, 0.8)',
+    },
+  },
+  variants: {
+    'dashboard-plans': {
+      dialogContainer: {
+        minW: '100vw',
+        minH: '100vh',
+      },
+      dialog: {
+        borderRadius: 0,
+        m: 0,
+        px: {
+          base: '10px',
+          sm: '20px',
+        },
+        minW: '100vw',
+        minH: '100vh',
+        bgColor: '#546E39',
+      },
+      header: {
+        pt: '50px',
+        textAlign: 'center',
+        color: 'white',
+        fontSize: '2xl',
+      },
+      body: {
+        px: 0,
+        pt: '30px',
+        display: 'flex',
+        justifyContent: { base: 'start', lg: 'center' },
+        alignItems: 'start',
+        gap: '10px',
+        overflowX: 'auto',
+      },
+      footer: {
+        pt: '50px',
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '20px',
+        justifyContent: 'center',
+        color: 'white',
+
+        '& > div:first-of-type': {
+          display: 'flex',
+          flexDirection: { base: 'column', md: 'row' },
+          justifyContent: 'center',
+          alignItems: { base: 'start', md: 'center' },
+          gap: '10px',
+
+          '& p': {
+            textAlign: 'center',
+            maxW: '800px',
+            fontWeight: 'bold',
+            whiteSpace: { md: 'nowrap' },
+          },
+        },
+        '& > p': {
+          textAlign: 'center',
+          maxW: '800px',
+          fontWeight: 'normal',
+        },
+
+        '& > div:nth-of-type(2)': {
+          display: 'flex',
+          alignItems: 'center',
+          gap: '10px',
+
+          '& > p': {
+            whiteSpace: 'nowrap',
+          },
+        },
+        '& > div:last-of-type': {
```

Comment:
In general, I don't think that do this is a good practice (styling the child using this kinds of css selectors). Maybe @elboletaire can suggest a better way to do this? Or we could just ignore it and go forward? 

Maybe creating a component and style the inner children it via props is a better approach?

Reply:
if I'm not mistaken, when creating a variant (of the modal in this case), we have to do it this way, since the parts of a modal (header, body, etc.) only extend so far, and to style the internal components, we need to access them in this way.

Reply:
Ideally, we should be using chakra-ui theming system (with multipart and normal). 

So rather than using `& > p` ideally we should define a section for that theme, and style it with the multi-part system, by name.

This will also help us in the future improve the theme and find issues, that otherwise would be hard to find due to the excess of CSS traversing.

Reply:
To do not block this PR anymore I preffer to accept it and from now try to do this as you said, using the multipart component.

I am going to mark this as resolved